python/cugraph/cugraph/tests/test_graph.py

```diff
@@ -739,3 +739,19 @@ def test_create_graph_with_edge_ids(graph_file):
     H = G.to_undirected()
     assert G.is_directed()
     assert not H.is_directed()
+
+
+# Test
+@pytest.mark.parametrize("graph_file", utils.DATASETS)
+def test_density(graph_file):
```

Comment:
Might want to also add a check that `MultiGraph` instances raise `TypeError` if their `density()` method is called.

Reply:
added test